### DISCUSSION TITLE
如何启动多个pcl2程序,而非单例

### DISCUSSION THREAD

**Discussion by yzqdev** [Meloong-Git/PCL#5094] [提问] [answered]
如题,每次点击pcl2程序,只有一个窗口,一个实例,个人需要开启多个实例用来对比版本隔离的整合包,如何开启多个pcl2实例,希望大佬们可以指点一二

**Comment by Joker2184** [comment, marked as answer]
默认只能开启一个无论版本

点快点能卡出来 两个版本

**Reply by yzqdev** [reply]
啊这

**Comment by hejiehao** [comment]
直接在 PCL 里再开一个游戏

**Comment by Ad-closeNN** [comment]
~~[https://multiple-pcl2.pages.dev/](https://multiple-pcl2.pages.dev/)~~

**Reply by yzqdev** [reply]
成功了,谢谢大佬!
